modules/rubiconBidAdapter.js

```diff
@@ -153,11 +153,9 @@ export const spec = {
 
       data.slots.push(slotData);
 
-      if (bidderRequest.gdprConsent) {
+      if (bidderRequest.gdprConsent && typeof bidderRequest.gdprConsent.gdprApplies === 'boolean' && bidderRequest.gdprConsent.gdprApplies === true) {
```

Comment:
You can skip the type check since you are doing the `===` in the third part of the `if` clause